docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst

```diff
@@ -12,14 +12,10 @@ Release notes: Open GEE 5.3.6
 

```

Comment:
I think we should remove the above line ("Open GEE 5.3.6 is currently in development"). Technically it's still true but we don't want to have to make another commit after we finish testing.